src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/IdeTestAssemblyRunner.cs

```diff
@@ -65,6 +63,7 @@ protected override async Task<RunSummary> RunTestCollectionAsync(IMessageBus mes
                             var test = new XunitTest(testCase, testCase.DisplayName);
                             ExecutionMessageSink.OnMessage(new TestStarting(test));
                             ExecutionMessageSink.OnMessage(new TestFailed(test, 0, null, new InvalidOperationException("Test did not run due to a harness failure.", ex)));
+                            result.Failed++;
```

Comment:
📝 This fixes a failure to report harness errors as test failures.
https://dev.azure.com/dnceng/public/_build/results?buildId=1256507

```
failed="0"
```